app/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,153 @@
+    <string name="available_to_be_mentor_mentee">Dostępne jako:</string>
+    <string name="settings">Ustawienia</string>
+    <string name="error_name_too_short">Nazwa powinna zawierać co najmniej %1$d znaków</string>
+    <string name="error_name_too_long">Nazwa powinna mieć maksymalnie %1$d znaków</string>
+    <string name="model_task_1">Dodaj sekcję Projekty do CV</string>
+    <string name="model_task_2">Przygotuj pytania do rozmowy kwalifikacyjnej X</string>
+    <string name="model_task_3">Udostępnij aktualizację statusu na Twitterze</string>
+    <string name="details">Szczegóły</string>
+    <string name="tasks">Zadania</string>
+    <string name="add_new_task">Dodaj nowe zadanie</string>
+    <string name="fetching_current_relation">Pobieranie informacji o bieżącej relacji…</string>
+    <string name="pending_requests">Oczekujące wnioski</string>
+    <string name="accepted_requests">Zaakceptowane wnioski</string>
+    <string name="rejected_requests">Odrzucone wnioski</string>
+    <string name="completed_relations">Ukończone relacje</string>
+    <string name="welcome">Witamy</string>
+    <string name="no_recent_achievements">Brak ostatnich osiągnięć</string>
+    <string name="recent_achievements">Ostatnie osiągnięcia</string>
+    <string name="confirm_logout">Potwierdź wylogowanie</string>
+    <string name="confirm_logout_msg">Czy na pewno chcesz się wylogować?</string>
+    <string name="logout">Wyloguj</string>
+    <string name="confirm_cancel_relation">tak</string>
+    <string name="cancel_relation_text">Czy na pewno chcesz anulować relację?</string>
+    <string name="cancel_relation_title">Anuluj relację</string>
+    <string name="empty_members_list">Nie ma innych członków.</string>
+    <string name="cancel_relation_denied">Nie</string>
+    <string name="current_password">Aktualne hasło</string>
+    <string name="new_password">Nowe hasło</string>
+    <string name="password_not_match">Hasła nie pasują do siebie</string>
+    <string name="ok">Dobrze</string>
+    <string name="change_password">Zmień hasło</string>
+    <string name="password_updated">Hasło zostało zmienione!</string>
+    <string name="current_new_password_should_not_same">Aktualne i nowe hasła nie powinny być takie same</string>
+    <string name="about">O</string>
+    <string name="about_systers">Systers to międzynarodowa społeczność dla wszystkich kobiet zaangażowanych w techniczne aspekty informatyki. Z zadowoleniem przyjmujemy udział kobiet technologów w każdym wieku i na każdym etapie ich studiów lub kariery. Angażujemy naszą społeczność, pomagając w rozwoju oprogramowania open source, współpracując ze społecznością globalną oraz ucząc się i doskonaląc nasze umiejętności programistyczne. Naszym celem jest zapewnienie bezpiecznej, pozytywnej społeczności online dla wielu naszych wolontariuszy, którzy oferują swoje umiejętności, czas i zaangażowanie w nasze projekty.</string>
```

Comment:
Systers to międzynarodowa społeczność dla wszystkich kobiet zaangażowanych w techniczne aspekty informatyki. Z radością przyjmujemy nowe członkinie w każdym wieku i na każdym etapie nauki lub kariery. Angażujemy naszą społeczność, pomagając w rozwoju oprogramowania open source, współpracując ze społecznością globalną oraz ucząc się i doskonaląc nasze umiejętności programistyczne. Naszym celem jest zapewnienie bezpiecznej, pozytywnej społeczności online dla wielu naszych wolontariuszy, którzy oferują swoje umiejętności, czas i zaangażowanie w nasze projekty.